.github/workflows/ci.yml

```diff
@@ -113,6 +113,9 @@ jobs:
       - name: Install dependencies
         run: yarn install --immutable
 
+      - name: Dependencies are consistent
+        run: yarn syncpack list-mismatches
```

Comment:
The downside to workspaces in a monrepo specifying their own dependencies is (a) repetition, and (b) things might get out of sync.

If workspace1 specifies, say, `axios^1.0.0` and workspace2 specifies `axios^0.27.1`, then Node can satisfy the dependencies by installing two separate versions of `axios`. That *can* be appropriate, if we really need two versions or are incrementally updating for some reasons. But in general, if two workspaces use the same dependency, we want the versions to match[^1]. `syncpack` is a tool for enforcing this. It's very configurable, so if we wanted intentional version differences, we could allow that. Our current usage does not require any configuration.

Note: When [Renovate](https://github.com/renovatebot/renovate) updates dependencies, it will update all the package.jsons together, so syncpack will still be happy. 

[^1]: At the very least we want the versions to be compatible, but having them match is easiest.